react/components/molecules/card/card.stories.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { storiesOf } from "@storybook/react-native";
+import { withKnobs, select, text } from "@storybook/addon-knobs";
+import { View } from "react-native";

```

Comment:
Nitpicking again, but move this below "react" and before "storybook" imports pls